### ISSUE TITLE
Febrile Seizures, Familial

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/monarch-disease-ontology-RETIRED#33]
_From @cmungall on March 27, 2015 13:27_

Compare

```
[Term]
id: OMIM:604403
name: Generalized Epilepsy With Febrile Seizures Plus, Type 2
is_a: DC:0000159 ! Febrile Seizures, Familial
is_a: DC:0000160 ! Epilepsy, Generalized, with Febrile Seizures Plus
```

With

```
[Term]
id: OMIM:604233
name: Generalized Epilepsy With Febrile Seizures Plus, Type 1
is_a: DC:0000160 ! Epilepsy, Generalized, with Febrile Seizures Plus
```

Is this intentional, or is this just lack of consistency over how restrictive the parent (DC:0000159 ) is?

_Copied from original issue: monarch-initiative/human-disease-ontology#24_

**Comment by nicolevasilevsky** [comment]
_From @pnrobinson on March 27, 2015 13:44_

If you read the description of the diseases of these two OMIM entries, the classification is justified. Admittedly there is a lot of overlap, but only the first entry seems to be associated with classical familial febrile seizures.
-Peter

Dr. med. Peter N. Robinson, MSc.
Professor of Medical Genomics
Professor in the Bioinformatics Division of the Department of Mathematics and Computer Science of the Freie Universit�t Berlin
Institut f�r Medizinische Genetik und Humangenetik
Charit� - Universit�tsmedizin Berlin
Augustenburger Platz 1
13353 Berlin
Germany
+4930 450566006
Mobile: 0160 93769872
peter.robinson@charite.de
http://compbio.charite.de
http://www.human-phenotype-ontology.org
Introduction to Bio-Ontologies: http://www.crcpress.com/product/isbn/9781439836651
I have learned from my mistakes, and I am sure I can repeat them exactly
ORCID ID:http://orcid.org/0000-0002-0736-9199
Scopus Author ID 7403719646
Appointment request: http://doodle.com/pnrobinson
________________________________________
Von: Chris Mungall [notifications@github.com]
Gesendet: Freitag, 27. M�rz 2015 14:27
An: monarch-initiative/human-disease-ontology
Betreff: [human-disease-ontology] Febrile Seizures, Familial (#24)

Compare

[Term]
id: OMIM:604403
name: Generalized Epilepsy With Febrile Seizures Plus, Type 2
is_a: DC:0000159 ! Febrile Seizures, Familial
is_a: DC:0000160 ! Epilepsy, Generalized, with Febrile Seizures Plus


With

[Term]
id: OMIM:604233
name: Generalized Epilepsy With Febrile Seizures Plus, Type 1
is_a: DC:0000160 ! Epilepsy, Generalized, with Febrile Seizures Plus


Is this intentional, or is this just lack of consistency over how restrictive the parent (DC:0000159 ) is?

�

**Comment by nicolevasilevsky** [comment]
_From @sbello on March 27, 2015 14:40_

This was based entirely on the way OMIM placed the 2 terms in their phenotypic series. OMIM has 604403 in both series and 604233 only in PS604233.
Sue

**Comment by nicolevasilevsky** [comment]
_From @cmungall on March 27, 2015 17:55_

OK, great, just checking.

Fixes #24
